.github/workflows/coverity.yml

```diff
@@ -26,14 +26,16 @@ jobs:
         COVERITY_TOKEN: ${{ secrets.COVERITY_TOKEN }}
 
     - name: Prepare
+      uses: awalsh128/cache-apt-pkgs-action@latest
+      with:
+        packages: libevent-dev cmake
+        version: 1.0
+
+    - name: Build with Coverity
       run: |
-        sudo apt-get install -y libevent-dev gcc cmake
```

Comment:
Missing gcc?

Reply:
Since gcc is already installed on githubs ubuntu I removed this.
A working gcc should be good enough for Coverity.